src/layers/invertible_layer_conv1x1.jl

```diff
@@ -127,6 +127,11 @@ function conv1x1_grad_v(X::AbstractArray{T, N}, ΔY::AbstractArray{T, N},
     dv2 = cuzeros(X, k)
     dv3 = cuzeros(X, k)
 
+    # Do not calculate gradients if layer is frozen
+    if C.freeze 
+        return dv1, dv2, dv3 
```

Comment:
Isn't there a `Zero` type that is basically memory free in Flux for these things

Reply:
Im not really sure but I think they dont support it anymore:
https://github.com/FluxML/Flux.jl/blob/master/src/deprecations.jl#L29

Reply:
ok